vivisect/base.py

```diff
@@ -229,7 +229,9 @@ def _handleADDRELOC(self, einfo):
             if ptr != (ptr & e_bits.u_maxes[self.psize]):
                 logger.warn('RTYPE_BASERELOC calculated a bad pointer: 0x%x (imgbase: 0x%x)', ptr, imgbase)
 
+            self._supervisor = True
```

Comment:
Instead of doing this, if self._supervisor is going to be a thing going forward, we can add a function to do this:
```
@contextlib.contextmanager
def getAdminRights(self):
    self._supervisor = True
    yield
    self._supervisor = False
``` 
(and import contextlib at the top)
And then use it like this:
```
with self.getAdminRights():
    self.writeMemoryPtr(rva, ptr)
```

Reply:
sorry, didn't see this.  trying it instead of my object-crazy method.